easybuild/main.py

```diff
@@ -223,8 +223,6 @@ def main(args=None, logfile=None, do_build=None, testing=False):
         # print location to last log file, and exit
         last_log = find_last_log(logfile) or '(none)'
         print_msg(last_log, log=_log, prefix=False)
-        cleanup(logfile, eb_tmpdir, testing, silent=True)
```

Comment:
so, this changes functionally? You can now combine `--last-log` with `--search`?